runtime/src/accounts.rs

```diff
@@ -230,6 +234,21 @@ impl Accounts {
         })
     }
 
+    fn load_sysvar_account(
+        &self,
+        sysvar_type: &SysvarType,
+        pubkey: &Pubkey,
```

Comment:
That thought did cross my mind halfway through adding all of these conversions.  Since it's now part of the `Sysvar` trait, it *should* be tough to screw this up, and it'll just add a few more lines when adding a new sysvar. With that said, I'm not leaning strongly either way on this, @jstarry thoughts?